extensions/frontend-tiles/src/integration-test/TestContext.ts

```diff
@@ -0,0 +1,57 @@
+import { AccessToken } from "@itwin/core-bentley";
+import { getAccessTokenFromBackend } from "@itwin/oidc-signin-tool/lib/cjs/frontend";
+
+export class TestContext {
+  private _accessToken: AccessToken | undefined;
+  private static _instance?: TestContext = undefined;
+
+  private constructor() { }
+
+  public static async instance(): Promise<TestContext> {
+
+    if (this._instance === undefined) {
+      this._instance = new TestContext();
+      await this._instance.initialize();
+    }
+    return this._instance;
+  }
+
+  public getAccessToken(): AccessToken | undefined {
+    return this._accessToken;
+  }
+
+  private async initialize() {
+    const oidcConfig = {
+      clientId: process.env.MES_OIDC_CLIENT_ID || "",
+      redirectUri: process.env.OIDC_REDIRECT || "",
+      scope: process.env.MES_OIDC_SCOPE || "",
+      authority: process.env.MES_OIDC_AUTH_URL || "",
+    };
+
+    const user = {
+      email: process.env.mes_integration_testuser || "",
```

Comment:
why are these env vars all lowercase vs the others above which are all capitalized?